tests/server/disco/TestViews.js

```diff
@@ -33,17 +33,27 @@ describe('Discovery Pane GET requests', () => {
       assert.include(policy.get('connect-src'), 'https://addons.mozilla.org');
     }));
 
-  it('should be using SRI for script and style in /', () => request(app)
+  it('should be using SRI for script and style', () => request(app)
     .get(defaultURL)
     .expect(200)
     .then((res) => checkSRI(res)));
 
-  it('should be a 404 for requests to /', () => request(app)
+  it('will redir to /en-US/firefox/  for requests to /', () => request(app)
```

Comment:
Yeah I've been thinking the same thing - I'd like to try and kill as many of these crufty server tests off as possible. I was thinking that it might be more direct to see about testing the CSP config via the CSP in a similar way to the way this middleware is tested. That will then protect against the issue of the CSP middleware changing underneath us was originally the main concern.